inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterInfo.java

```diff
@@ -37,6 +42,11 @@
 @AllArgsConstructor
 @ApiModel("Inlong cluster info")
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, visible = true, property = "type")
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = DataProxyClusterInfo.class, name = ClusterType.DATAPROXY),
```

Comment:
Please use the custom `JsonTypeDefine` annotation, which is more suitable for this scenario that supports externally extended subclasses.

Reply:
JsonTypeDefine is used in child class, this is used in parent class, what do you mean?